lib/stat_tracker.rb

```diff
@@ -1,113 +1,36 @@
 require "csv"
+require "./lib/games_statistics_collection"
+require "./lib/game_teams_statistics_collection"
+require "./lib/teams_statistics_collection"
 
 class StatTracker
   attr_reader :games,
               :teams,
               :game_teams
 
-  def initialize(file_paths)
-    @games = file_paths[:games]
-    @teams = file_paths[:teams]
-    @game_teams = file_paths[:game_teams]
+  def initialize
+    @games = games_collection
+    @teams = teams_collection
+    @game_teams = game_teams_collection
   end
 
   def self.from_csv(file_path_locations)
-    self.new(file_path_locations)
+    @@games_path = file_path_locations[:games]
+    @@teams_path = file_path_locations[:teams]
+    @@game_teams_path = file_path_locations[:game_teams]
+    self.new
   end
 
-  # League Statistics
-  def count_of_teams
-    total_games = CSV.read(@teams, :headers=>true)
-    total_games.count
+  def games_collection
```

Comment:
I think calling these all in one method would be easier, but we can talk that out.